src/OpenTelemetry.Instrumentation.SqlClient/SqlClientInstrumentationOptions.cs

```diff
@@ -39,6 +39,17 @@ public class SqlClientInstrumentationOptions
         private static readonly Regex DataSourceRegex = new Regex("^(.*?)\\s*(?:[\\\\,]|$)\\s*(.*?)\\s*(?:,|$)\\s*(.*)$", RegexOptions.Compiled);
         private static readonly ConcurrentDictionary<string, SqlConnectionDetails> ConnectionDetailCache = new ConcurrentDictionary<string, SqlConnectionDetails>(StringComparer.OrdinalIgnoreCase);
 
+        // .NET Framework implementation uses SqlEventSource from which we can't reliably distinguish
+        // StoredProcedures from regular Text sql commands.
+#if NETFRAMEWORK
+
+        /// <summary>
+        /// Gets or sets a value indicating whether or not the <see cref="SqlClientInstrumentation"/> should
+        /// add the text of the executed Sql commands as the <see cref="SemanticConventions.AttributeDbStatement"/> tag.
+        /// Default value: True.
+        /// </summary>
+        public bool SetStatementText { get; set; } = true;
```

Comment:
Lets keep the default `false`. Its a bit risky to collect full sql statement by default.

Reply:
Also - we can make it more obvious here that the statement text could be just storedproc name or the actual sql query.